scripts/plugins/extauth_hook_ad.py

```diff
@@ -0,0 +1 @@
+extauth-hook-AD.py
```

Comment:
why is this new file necessary?

Reply:
This is for unittest,  and it will not be installed into dom0

The file name contains `-`, which is not supported by module names.